policies/templates/gcp_vpc_sc_ip_range_v1.yaml

```diff
@@ -28,8 +28,8 @@ spec:
             maximum_cidr_size:
               type: integer
               description: "Maximum size in CIDR notation for an IP network"
-              minimum: 0
```

Comment:
Why aren't these supported?

Reply:
I believe it's a bug in the K8S yaml parser.